example.env

```diff
@@ -2,3 +2,9 @@ OWNER="yournamehere"
 IDENTIFIER="my-development-123"
 AWS_DEFAULT_REGION="us-west-2"
 AWS_DEFAULT_PROFILE="my-aws-account"
+RUNNER_TOKEN_SECRET_ARN="arn:aws:secretsmanager:us-west-2:1234567890:secret:pangeo-forge-aws-bakery/runner-token/my-token"
+PREFECT__CLOUD__AUTH_TOKEN="my-prefect-tenant-token"
+PREFECT_PROJECT="pangeo-forge-aws-bakery"
+PREFECT_AGENT_LABELS='["dask_test"]'
+PYTHONPATH=.
```

Comment:
@sharkinsspatial is this to address your import funkiness?

Reply:
Yes.